public/app/features/correlations/Forms/QueryEditorField.test.tsx

```diff
@@ -63,4 +81,97 @@ describe('QueryEditorField', () => {
       await screen.findByRole('alert', { name: 'Data source does not export a query editor.' })
     ).toBeInTheDocument();
   });
+
+  describe('Query validation', () => {
+    beforeEach(() => {
+      // Request errors cause the query runner to print to console,
+      // this means the test would fail because of the fail on log rules.
+      // We mock console.error to avoid this.
+      jest.spyOn(console, 'error').mockImplementation();
```

Comment:
I don't see any errors if I remove this bit (or change to `jest.spyOn(console, 'error').mockImplementation(() => { throw new Error() });`) so it doesn't seem like it's being called. Can we remove it?

Reply:
I removed it as it was leftover from the test improvements.